pinc/CharSuites.inc

```diff
@@ -83,6 +83,13 @@ class CharSuite
         $enabled_sets = CharSuites::get_enabled();
         return in_array($this, $enabled_sets);
     }
+
+    // Validate that combined characters in the charsuite are normalized
```

Comment:
I can't connect to unicode.org at the moment to check but I seem to remember there could be single codepoints that normalise to others such as Å. If this is so we could change this comment to leave out combined.

Reply:
You are absolutely correct and we should be checking for all the characters in the range, not just specific points. I've updated the code accordingly.